thefuck/rules/dnf_no_such_command.py

```diff
@@ -30,8 +30,11 @@ def _get_operations():
 
 @sudo_support
 def get_new_command(command):
-    misspelled_command = regex.findall(command.output)[0]
-    return replace_command(command, misspelled_command, _get_operations())
+    if misspelled_command  == 'uninstall':
+        return [command.script.replace('uninstall','remove')
+    else:
+        misspelled_command = regex.findall(command.output)[0]
+        return replace_command(command, misspelled_command, _get_operations())
```

Comment:
👉 **[Nit](https://google.github.io/eng-practices/review/reviewer/standard.html)**: An `else` statement right after a `return` is unnecessary and can be removed. That increases readability and improves maintainability. So, this change should actually read as the following:

```diff
 @sudo_support
 def get_new_command(command):
+    if misspelled_command  == 'uninstall':
+        return [command.script.replace('uninstall','remove')
     misspelled_command = regex.findall(command.output)[0]
     return replace_command(command, misspelled_command, _get_operations())
```